src/tests/simple.rs

```diff
@@ -52,3 +52,56 @@ fn basics() {
         None
     );
 }
+
+#[test]
+fn get_changes() {
+    let tempdir = tempfile::tempdir().unwrap();
+    let db = create_rocks_db(tempdir.path()).unwrap();
```

Comment:
We used it because at start we agreed that your main usage will be with RocksDB and so we want to test with this DB to be close to your usecase. Can be changed but might need to verify the hashmap_db